src/Controller/AuthenticationController.php

```diff
@@ -60,9 +60,14 @@ public function authenticateAction(Request $request, int $version): Response
                 throw new \RuntimeException('Invalid instance given');
             }
 
+            $customer = $user->getCustomer();
+            if (!$customer) {
+                return new JsonResponse(null, 200, []);
```

Comment:
no sure where this api is called, but isnt this an error?

Reply:
have taken a deeper look into - the correct status code would be 401 (will change it) - but i think it will never happen but its nullable